### ISSUE TITLE
Not loading in IE9+ 

### ISSUE THREAD

**Issue by HendrikThePendric** [mozilla/localForage#410]
I simply can’t get the localForage to load in any version of IE 9+. I seem to have the exact same issue as [issue 323](https://github.com/mozilla/localForage/issues/323) (`Object doesn't support this action` on line 2585 of localforage.js). However, this issue is quite old and seems to have been resolved in December 2014. I am not using any type of build system and am simply downloading the file `localforage.js` from the [1.2.4 release page](https://github.com/mozilla/localForage/releases/tag/1.2.4).

I have also tried [release 1.1.1](https://github.com/mozilla/localForage/releases/tag/1.1.1) but the issue remains.

My initial thought was that I am using the wrong file or something but @tofumatt suggested I filed an issue, so here I am....

**Comment by thgreasi** [comment]
What is the full path of the file they you are using?
Does the same file work on other browsers?

**Comment by HendrikThePendric** [comment]
Sorry, my bug report is over a month old now and the project I was working on is actually finished. However, I did just try to replicate the steps I took back then with version 1.2.10 (as opposed to 1.2.4). This time all seems to work fine.

So, I don't know if something was wrong with the hosted code, or I was causing an issue on my end. In any case there is no issue anymore now. I will close it.
